NavigationReactNative/src/ios/NVSceneController.m

```diff
@@ -47,14 +46,23 @@ - (void)viewWillAppear:(BOOL)animated
         [self.navigationItem setTitle:navigationBar.title];
     }
 
-    [navigationBar updateColors];
+    [navigationBar updateTintColors];
+    [navigationBar updateTitleColors];
 
     if (@available(iOS 11.0, *)) {
         self.navigationController.navigationBar.prefersLargeTitles = true;
         [self.navigationItem setLargeTitleDisplayMode:navigationBar.largeTitle ? UINavigationItemLargeTitleDisplayModeAlways : UINavigationItemLargeTitleDisplayModeNever];
     }
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+    // This is seemingly required, otherwise the small navbar title color does not get reset when only called from viewWillAppear
```

Comment:
Thanks but I'm not a fan of code comments. This is great detail though so you can add it to the PR's description